### PR TITLE
include protocol_version for the default target group

### PR DESCRIPTION
**Pull request by azarudeena**
## what
* ability to define the protocol version for the target groups. 
* made in this sync with the [alb](https://github.com/cloudposse/terraform-aws-alb/blob/master/variables.tf#L235-L239) module 

## why
* Need to have a case for GRPC service and health-checks were failing. 

## references
* AWS ELB [Health Check](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/target-group-health-checks.html)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -121,6 +121,12 @@ variable "protocol" {
   description = "The protocol for the created ALB target group (if `target_group_arn` is not set)"
 }
 
+variable "protocol_version" {
+  type        = string
+  default     = "HTTP1"
+  description = "The protocol_version for the created ALB target group (if `target_group_arn` is not set and if protocol is either HTTP or HTTPS)"
```
Comment (nitrocode) [inline variables.tf]: Can we use the same description as the upstream provider docs

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#protocol_version

Comment (nitrocode) [inline variables.tf]:
```suggestion
  description = "Only applicable when protocol is `HTTP` or `HTTPS`. The protocol version. Specify GRPC to send requests to targets using gRPC. Specify HTTP2 to send requests to targets using HTTP/2. The default is `HTTP1`, which sends requests to targets using HTTP/1.1"
```

Comment (azarudeena) [reply variables.tf]: Updated the decription as requested.

**Review by nitrocode — APPROVED** [review]

**Pull request merged**